content/tutorials/dev-guide.md

```diff
@@ -28,37 +21,42 @@ Our [User Guide](user-guide) will give you some pointers for navigation and vide
 
 The easiest way to begin programming in the ARENA is to install the [Python library](../python) and create your first Python program. ARENA communications are a series of MQTT messages which govern all objects and their properties. This library is a wrapper which will allow you to easily send and receive those messages.
 
-## Create a cube and observe
+## Create a box and observe
 
-Now, create the simplest Python program you can below, which will generate a 1-meter cube. By default, objects are generated in white (#FFFFFF), with no rotation, at scene x, y, z position (0, 0, 0), and with no other properties applied. Some of the other properties you can add to objects are detailed in our [Python Examples](../python/objects). Notice that cube seems stuck in the ground, which is due to the cube's origin at its center positioned at scene coordinates (0, 0, 0). If you enable Flying mode (see above), you can move below the ground plane and view the other half of the cube. Type Ctrl-C to end the program.
+Now, create the simplest Python program you can below, which will generate a 1-meter box. By default, objects are generated in a gray color, with no rotation, at scene x, y, z position (0, 0, 0), and with no other properties applied. Some of the other properties you can add to objects are detailed in our [Python Examples](../python/objects). Notice that the box seems stuck in the ground, which is due to the box's origin at its center positioned at scene coordinates (0, 0, 0). If you enable Flying mode (see [User Guide](user-guide)), you can move below the ground plane and view the other half of the box. Type Ctrl-C to end the program.
```

Comment:
New aframe update randomizes colors, I believe. So objects are not gray by default if the color attribute is not specified in material.